scripts/handlers/subtract.js

```diff
@@ -12,7 +12,7 @@ function subtract(x, y) {
   if (typeof x !== 'number') { throw new TypeError('x'); }
   if (typeof y !== 'number') { throw new TypeError('y'); }
 
-  const result = _;
+  const result = x - y;
 
   if (typeof result !== 'number') { throw new TypeError('result'); }
   return result;
```

Comment:
Below this line there should be a "}" to close the subtract(x, y) function

Reply:
ok. Thanks